target/linux/ramips/mt7621/base-files/etc/init.d/set-irq-affinity

```diff
@@ -2,30 +2,20 @@
 
 START=99
 
-get_irq() {
-	local name="$1"
-	grep -m 1 "$name" /proc/interrupts | cut -d: -f1 | sed 's, *,,'
-}
-
-set_irq_affinity() {
-	local name="$1"
-	local val="$2"
-	local irq="$(get_irq "$name")"
-	[ -n "$irq" ] || return
-	echo "$val" > "/proc/irq/$irq/smp_affinity"
-}
-
 start() {
 	if grep -q 'processor.*: 2' /proc/cpuinfo; then
-		mask1=4
-		mask2=8
+		mask=4
 	elif grep -q 'processor.*: 1' /proc/cpuinfo; then
-		mask1=2
-		mask2=2
+		mask=2
 	else
 		return
 	fi
 
-	set_irq_affinity mt76x2e $mask1
-	set_irq_affinity mt7603e $mask2
+	for irq in $(grep "mt76..e" /proc/interrupts | cut -d: -f1 | sed 's, *,,')
```

Comment:
I don't really understand what this is about, but from the code it looks to me like order of interfaces can change here.
In the "old" version, mt76x2 always got mask1 and mt7603e got mask2.

In the for loop, the order of returned strings would be inverted if sorted, and should be random in the current "new" implementation. However, I don't know whether this is a problem for the actual use case.

Reply:
So all `mt76xxe` IRQs are moved to the 2nd CPU. Order does not matter.